### ISSUE TITLE
Compile.sh no longe is able to compile a single chapter

### ISSUE THREAD

**Issue by kilon** [SquareBracketAssociates/UpdatedPharoByExample#124]
looks like all actions have been delegated to pillarPostExport.sh but even that bash script is not able to compile a single chapter. This is very bad for those that want to compile fast small changes as its the most case scenarios.

**Comment by zio-pietro** [comment]
it's even worse. I found syntax for one chapter processing in Pillar manual:
./pillar export --to=latex <chapterPath> (path relative o absolute, it doesn't matter)
Pillar creates output.tex file in book-result dir. 
First: this file is full of ^M chars, even if I work on Unix-like
system. (The files produced by ./compile.sh are clean)
Second: For latex processing Pillar make pillarPostExport.sh file. But with this file nor by hand (invoking pdflatex or lualatex command from command line) the file output.tex can't be processed. Work finish with error.
./output.tex:9: Undefined control sequence
      \definecolor{codebox}{HTML}{e4e4e4}
Even comenting out line 9 the result don't change much. Now pillarPostExport ended with pdf file created, but with vanilla LaTeX settings, 'cose output.tex is incomplete, compared with .tex files produced by compile.sh. I found also in file output.tex, made with '--to=latex' option bit of text 'ampersand gt semicolon'. AFAIK it's HTML entity.
Never mind, for now I continue with complete run (compile.sh), having time for tobacco breaks ;)
And I must dissect Pillar for discover what I'm doing wrong (or maybe what Pillar does wrong, who knows?)
Found somewhere: The best way to lose voluntary collaborators is to equip them with the tools that do not work :)
Personally I don't see the Pillar choice as a win-win situation (but I'm not impartial, having used LaTeX for 15+ yrs)
